packages/thirdweb/src/exports/extensions/pack.ts

```diff
@@ -0,0 +1,8 @@
+export {
+  PACK_TOKEN_TYPE,
+  type ERC20Reward,
+  type ERC721Reward,
+  type ERC1155Reward,
+  type CreatePackParams,
+  createPack,
```

Comment:
dont we need openPack and getPackContents exported here too?

Reply:
oops mb

Reply:
wait, actually I remember why I didn't do it